src/Directory.Build.props

```diff
@@ -0,0 +1,19 @@
+<!-- Do not use XML namespaces here (<Project xmlns="...">), because it would break patching of version numbers during builds on AppVeyor (https://github.com/appveyor/website/pull/409) -->
+<Project>
+  <PropertyGroup>
+    <Authors>Pieter van Ginkel, Git Extensions</Authors>
+    <Company>Git Extensions</Company>
+    <RepositoryUrl>https://github.com/gitextensions/GitExtensions.GerritPlugin</RepositoryUrl>
+    <LangVersion>latest</LangVersion>
+    <Version>0.0.0</Version>
+    <AssemblyVersion>0.0.0.1</AssemblyVersion>
+    <FileVersion>0.0.0.1</FileVersion>
+    <InformationalVersion>0.0.0.1</InformationalVersion>
+    <PackageVersion>0.0.0.1</PackageVersion>
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <TargetFramework>net461</TargetFramework>
```

Comment:
Have a common Directory.Build.props in the root directory, where have common properties.
Like in GE: https://github.com/gitextensions/gitextensions/blob/master/Directory.Build.props

Reply:
You're right. 
Done.